client/src/components/payment/payment-summary.vue

```diff
@@ -50,107 +50,46 @@ import PaymentMixin from '@/components/payment/payment-mixin'
   }
 })
 export default class PaymentSummary extends Mixins(PaymentMixin) {
-  @Prop(Number) id: number
-  @Prop(Object) summary: any
-  @Prop(Object) receipt: any
-  @Prop({ default: {
-    receipt: {
-      created_on: '',
-      total: 0.00,
-      references: [{
-        receipt_number: '',
-        status_code: ''
-      }]
-    },
-    summary: {
-      completionDate: '',
-      statusCode: ''
-    }
-  } })
-  protected fetchError: string = ''
+  @Prop(Number)
+  readonly id: number
+
+  @Prop(Object)
+  readonly summary: any
+
+  @Prop(Object)
+  readonly receipt: any
+
+  protected fetchError = ''
+
+  private loading = false
 
   async downloadReceipt () {
     const { id } = this
+    this.loading = true
     await this.downloadReceiptPdf(id)
+    this.loading = false
   }
 }
 </script>
 
-<style lang='scss' scoped>
-@import '../../assets/scss/theme';
```

Comment:
I never saw the scss in the app until now. Certainly could be confusing when moving between components 😅 I know time is tight right now, but we should definitely track that we need to refactor the styling to scss app wide.

Reply:
https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/5730